pkg/standalone-utils/test/ProtocolIdRegistry.test.ts

```diff
@@ -99,4 +102,36 @@ describe('ProtocolIdRegistry', () => {
       expect(await registry.isValidProtocolId(MAX_UINT256)).to.equal(false);
     });
   });
+
+  describe('renaming protocol IDs', async () => {
+    const newProtocolName = 'Test Protocol';
+    let transactionReceipt: ContractReceipt;
+
+    context('authorized user', async () => {
+      sharedBeforeEach('rename protocol', async () => {
+        transactionReceipt = await (await registry.connect(authorizedUser).renameProtocolId(0, newProtocolName)).wait();
+      });
+
+      it('event emitted', async () => {
+        expectEvent.inReceipt(transactionReceipt, 'ProtocolIdRenamed', {
+          protocolId: 0,
+          name: newProtocolName,
+        });
+      });
+
+      it('successful renaming', async () => {
+        expect(await registry.getProtocolName(0)).is.equal(newProtocolName);
+      });
+    });
+
+    it('trying to rename a non-registered Id', async () => {
+      await expect(registry.connect(authorizedUser).renameProtocolId(MAX_UINT256, newProtocolName)).to.be.revertedWith(
+        'Protocol ID not registered'
+      );
+    });
+
+    it('Unauthorized user not able to rename an ID', async () => {
```

Comment:
I'd place this last one under a different context (e.g. `when the user is not authorized to rename`; `it('reverts')`)